### PR TITLE
added logging

### PR DESCRIPTION
**Pull request by VitaliySokoliuk**

### CODE REVIEW

**Comment by VitaliySokoliuk** [comment]
What went wrong with logging?

File: src/main/java/ua/lviv/services/ConnectionUtil.java [hunk 1]
```diff
@@ -3,22 +3,29 @@
...
 
 public class ConnectionUtil {
 
     private static final String JDBC_URL = "jdbc:mysql://localhost:3306/my_project?serverTimezone=UTC";
     private static final String USERNAME = "root";
     private static final String PASSWORD = "qwerty";
+    private static Logger LOG = Logger.getLogger(ConnectionUtil.class);
 
     public static Connection getConnection(){
+        DOMConfigurator.configure("loggerConfig.xml");
```
Comment (VolodymyrTymtsiv) [inline src/main/java/ua/lviv/services/ConnectionUtil.java]: Do not use ` DOMConfigurator.configure("loggerConfig.xml");` in each service.
Let's create a listener for the configuration that will be read once.
Example - https://github.com/VolodymyrTymtsiv/IShop/blob/master/src/main/java/ua/lviv/lgs/ContextListener.java

**Comment by VitaliySokoliuk** [comment]
Doesn`t work again

**Comment by VolodymyrTymtsiv** [comment]
Try to replace with `$PROJECT_DIR$/logs/error.log`

**Pull request merged**